pkg/network/tracer/ebpf_conntracker.go

```diff
@@ -470,6 +470,11 @@ func getManager(cfg *config.Config, buf io.ReaderAt, constants []manager.Constan
 }
 
 func getPrebuiltConntracker(cfg *config.Config) (bytecode.AssetReader, []manager.ConstantEditor, error) {
+	kv414 := kernel.VersionCode(4, 14, 0)
+	if kv, err := kernel.HostVersion(); err != nil || kv < kv414 {
```

Comment:
If `err != nil` its not because we don't support prebuilt conntracker? So the error reported would be incorrect in that case. These should be handled separately: one case for the error from `HostVersion` and another case for the kernel requirement.

Reply:
good idea will fix